.travis.yml

```diff
@@ -21,6 +21,7 @@ matrix:
     fast_finish: true
 
 before_script:
+  - composer install
```

Comment:
why is this needed?
it will basically do update here then anyway, as install without look file does this.